src/storage/storage.h

```diff
@@ -70,9 +70,10 @@ enum class ColumnFamilyID : uint32_t {
   Propagate,
   Stream,
   Search,
+  TDigest,
 };
 
-constexpr uint32_t kMaxColumnFamilyID = static_cast<uint32_t>(ColumnFamilyID::Search);
+constexpr uint32_t kMaxColumnFamilyID = static_cast<uint32_t>(ColumnFamilyID::TDigest);
```

Comment:
I think we can use `PrimarySubkey` to store subkeys instead of a new column family?